src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as

```diff
@@ -88,45 +92,19 @@ public class VideoMediaProvider extends MediaProvider {
         }
     }
 
-    /** Load new media file; only requested once per item. **/
-    override public function load(itm:PlaylistItem):void {
-        clearInterval(_interval);
-        _item = itm;
-        // Set Video or StageVideo
-        if (!_video) {
-            _video = new Video();
-            _video.smoothing = true;
-        }
-
-        _video.attachNetStream(_stream);
-
-        // Set initial quality and set levels
-        _currentQuality = 0;
-
-        for (var i:Number = 0; i < _item.levels.length; i++) {
-            if (_item.levels[i]["default"]) {
-                _currentQuality = i;
-                break;
-            }
-        }
-
-        if (config.qualitylabel) {
-            for (i = 0; i < _item.levels.length; i++) {
-                if (_item.levels[i].label == config.qualitylabel) {
-                    _currentQuality = i;
-                    break;
-                }
-            }
+    override public function init(itm:PlaylistItem):void {
+        setupVideo(itm);
+        // start loading the content only if preload is 'auto'
+        if (itm.preload === "auto") {
```

Comment:
Here `preload` can be any value that is not "none". We want to preload when item `preload` is set to "meta".